common/test/acceptance/tests/lms/test_problem_types.py

```diff
@@ -385,6 +400,131 @@ def test_partially_complete_notifications(self):
         self.problem_page.click_submit()
         self.problem_page.wait_partial_notification()
 
+    @ddt.data('correct', 'incorrect')
+    def test_reset_problem(self, correctness):
+        """
+        Scenario: I can reset a problem
+
+        Given I am viewing a problem with randomization: always and with reset button: on
+        And I answer a problem correct/incorrect
+        When I reset the problem
+        Then my answer is marked "unanswered"
+        And The problem displays a "blank" answer
+        """
+        self.answer_problem(correctness)
+        self.problem_page.click_submit()
+        self.problem_page.click_reset()
+        self.wait_for_status('unanswered')
+        self.assertTrue(self.is_status_true('unanswered'))
+
+
+@ddt.ddt
+class ChangingAnswerOfProblemTestMixin(object):
+
+    @ddt.data(['correct', '1/1 point (ungraded)'], ['incorrect', '0/1 point (ungraded)'])
+    @ddt.unpack
+    def test_checkbox_score_after_answer_and_reset(self, correctness, score):
+        """
+        Scenario: I can see my score on problem when I answer it and after I reset it
+
+        Given I am viewing problem
+        When I answer problem correct/incorrect
+        Then I should see a score
+        When I reset the problem
+        Then I should see a score of points possible: 1/1 point (ungraded) -- 0/1 point (ungraded)
+        """
+        self.answer_problem(correctness)
+        self.problem_page.click_submit()
+        self.assertEqual(self.problem_page.problem_progress_graded_value, score)
+        self.problem_page.click_reset()
+        self.assertEqual(self.problem_page.problem_progress_graded_value, '0/1 point (ungraded)')
+
+    @ddt.data(['correct', 'incorrect'], ['incorrect', 'correct'])
+    @ddt.unpack
+    def test_reset_correctness_after_changing_answer(self, initial_correctness, other_correctness):
+        """
+        Scenario: I can reset the correctness of a problem after changing my answer
+
+            Given I am viewing problem
+            Then my problem's answer is marked "unanswered"
+            When I answer problem inital correctness
+            And I input an answer as other correctness
+            Then my problem's answer is marked "unanswered"
+            And I reset the problem
+        """
+        self.wait_for_status("unanswered")
+        self.assertTrue(self.is_status_true("unanswered"))
+        self.answer_problem(initial_correctness)
+        self.problem_page.click_submit()
+
+        self.wait_for_status(initial_correctness)
+        self.assertTrue(self.is_status_true(initial_correctness))
+
+        self.answer_problem(other_correctness)
+        self.wait_for_status("unanswered")
+        self.assertTrue(self.is_status_true("unanswered"))
+        self.problem_page.click_reset()
```

Comment:
Makes sense to have an assertion after but the lettuce test did not have any assertion. 
I think the reset step should be removed rather than to add assertion because it is not making any good sense here.